src/sinks/sentry/metrics/sink.rs

```diff
@@ -23,25 +44,36 @@ impl StreamSink<Event> for SentryMetricsSink {
         let _guard = sentry::init(self.dsn);
 
         while let Some(metric) = input.next().await {
-            let name = metric.series().name().name.clone();
+            let series = metric.series();
+            let series_name = series.name();
+            let mut name = series_name.namespace.clone().unwrap_or_default();
```

Comment:
this part should fix `system.cpu.usage` showing up as `cpu.usage`

Reply:
Could also be a comment. 🙃